src/Components/SearchComponent.js

```diff
@@ -24,14 +34,37 @@ const SearchComponent =({listOfTeamMembers , setFilteredData , setIsSearched})=>
             onSubmit={
                 (e)=>{
                     e.preventDefault();
-                    const filteredData = searchTeamData(searchText,listOfTeamMembers,setIsSearched);
+                    const filteredData = searchTeamData(searchText,listOfTeamMembers,setIsSearched,cityName);
                     setFilteredData(filteredData);
                 }
             }>
-                <input id="searchBox" placeholder="search" 
+                <div className="search">
+                <input placeholder="search" 
                        value={searchText}
                        onChange={settingValueForSetSearchText}></input>
+                </div>
+                
+                <div className="search">
+                <select value={stateName} onChange = {
+                    (e)=>{setStateName(e.target.value); }
+                    }>
+                    {Object.keys(stateCityObj).map(state=><option key ={state}
```

Comment:
Add an option with "" value so that search filter with name can be used seprately without chossing state and cities also.